Tests/Clappr_Tests/Classes/Plugins/Core/MediaControl/MediaControlTests.swift

```diff
@@ -0,0 +1,510 @@
+
+            describe("#render") {
+                it("starts hidden") {
+                    let mediaControl = MediaControl()
+
+                    mediaControl.render()
+
+                    expect(mediaControl.isHidden).to(beTrue())
+                }
+
+                it("has black background with 60% of opacity") {
+                    let mediaControl = MediaControl()
+
+                    mediaControl.render()
+
+                    expect(mediaControl.view.backgroundColor).to(equal(UIColor.clapprBlack60Color()))
+                }
+
+                it("fills the superview") {
+                    let frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+                    let superview = UIView(frame: frame)
+                    let mediaControl = MediaControl()
+
+                    superview.addSubview(mediaControl)
+                    mediaControl.render()
+
+                    expect(superview.constraints.count).to(equal(4))
+                }
+
+                it("inflates the MediaControl xib in the view") {
+                    let mediaControl = MediaControl()
+
+                    mediaControl.render()
+
+                    expect(mediaControl.container).to(beAKindOf(MediaControlView.self))
+                    expect(mediaControl.view.subviews).to(contain(mediaControl.container))
+                }
+            }
+
+            describe("options") {
+                it("has the same options as the Core") {
+                    let options: Options = ["foo": "bar"]
+                    let core = Core(loader: Loader(), options: options)
+
+                    let mediaControl = MediaControl(context: core)
+
+                    expect(mediaControl.options).toNot(beNil())
+                    expect((mediaControl.options!["foo"] as! String)).to(equal("bar"))
+                }
+            }
+
+            describe("plugins") {
+                it("has the list of plugins that comes from options") {
+                    let options: Options = [kMediaControlPlugins: [MediaControlPluginMock.self]]
+                    let core = Core(loader: Loader(), options: options)
+
+                    let mediaControl = MediaControl(context: core)
+                    mediaControl.render()
+
+                    expect(mediaControl.plugins.count).to(equal(1))
+                    expect(mediaControl.plugins[0]).to(beAKindOf(MediaControlPluginMock.self))
+                }
+            }
+
+            describe("Events") {
+                var coreStub: CoreStub!
+                var mediaControl: MediaControl!
+
+                beforeEach {
+                    coreStub = CoreStub()
+
+                    mediaControl = MediaControl(context: coreStub)
+                    mediaControl.animationDuration = 0.1
+                    mediaControl.secondsToHideControlFast = 0.1
+                    mediaControl.secondsToHideControlSlow = 0.1
+                    mediaControl.render()
+                }
+                /*
```

Comment:
We can delete these lines.

It doesn't make sense to leave it here. I don't think we will have any "settings" plugin.

Reply:
sure. 

=]